sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -163,4 +215,22 @@ public static SensitivityAnalysisResult run(Network network,
                                                 ContingenciesProvider contingenciesProvider) {
         return find().run(network, factorsProvider, contingenciesProvider);
     }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                String workingStateId,
+                                                SensitivityAnalysisParameters parameters,
+                                                SensitivityFactorsProvider factorsProvider) {
+        return find().run(network, workingStateId, parameters, factorsProvider);
+    }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                SensitivityAnalysisParameters parameters,
+                                                SensitivityFactorsProvider factorsProvider) {
+        return find().run(network, parameters, factorsProvider);
+    }
```

Comment:
Same remark about the parameter order. It seems that all the runAsync methods are missing.

@murgeyseb Do we add them all? Only some of them? I think we should be symmetrical and provide only part of them (the most useful) but in sync/async modes

Reply:
It should definitely be symmetrical between async and sync.